docs/api/web-frame-main.md

```diff
@@ -66,6 +66,15 @@ These methods can be accessed from the `webFrameMain` module:
 Returns `WebFrameMain | undefined` - A frame with the given process and routing IDs,
 or `undefined` if there is no WebFrameMain associated with the given IDs.
 
+### `webFrameMain.fromFrameToken(processId, frameToken)`
+
+* `processId` Integer - An `Integer` representing the internal ID of the process which owns the frame.
+* `frameToken` string - A `string` representing the unique frame token in the
```

Comment:
docs: i like the additional description we have on the `webFrame.findFrameByToken` API description:

> Frame tokens can be retrieved from WebFrame instances (webFrame.frameToken) and can also be retrieved from WebFrameMain instances using webFrameMain.frameToken.

ref https://github.com/electron/electron/blob/9a979477b339b14d0a9f1dcce5d715e0121037ea/docs/api/web-frame.md#webframefindframebytokenframetoken

Reply:
Updated to reference the equivalent renderer webFrame.frameToken property in [74af5f5](https://github.com/electron/electron/pull/47850/commits/74af5f53f82a69d1dea346de8d309e58de855f8d)